amazon-kinesis-client/src/test/java/software/amazon/kinesis/integration_tests/KCL2XIntegrationTest.java

```diff
@@ -0,0 +1,35 @@
+package software.amazon.kinesis.integration_tests;
```

Comment:
FYI: convention is to have IT intermixed w/ non-IT tests, and named appropriately. We already have the `pom.xml` configured w/ the expected suffix:
https://github.com/awslabs/amazon-kinesis-client/blob/master/amazon-kinesis-client/pom.xml#L218-L220